pymongo/asynchronous/bulk.py

```diff
@@ -87,7 +87,7 @@ def __init__(
         self,
         collection: AsyncCollection[_DocumentType],
         ordered: bool,
-        bypass_document_validation: bool,
+        bypass_document_validation: Optional[bool] = None,
```

Comment:
This should be `bypass_document_validation: Optional[bool],` the arg should still be required.